### PR TITLE
Ad response meta data

### PR DESCRIPTION
**Pull request by dwick**
not ready for review

### CODE REVIEW

**Comment by dwick** [comment]
updated event specs: https://reddit.atlassian.net/wiki/pages/diffpagesbyversion.action?pageId=36503645&selectedPageVersions=6&selectedPageVersions=5

**Comment by dwick** [comment]
relevant tickets:

f86c4a6: https://reddit.atlassian.net/browse/ADS-317
e3f9ee3: https://reddit.atlassian.net/browse/ADS-390

No ticket for adding the priority id.  I thought i could possibly be able to remove this query: https://github.com/reddit/reddit-public/commit/5f32e1032a09fbe2b6c4eb955200f80684a492f3#diff-af03c6f59a55db9d7946ad8013d50ffcR618.  I also think the data is generally useful in events.

**Comment by dwick** [comment]
:eyeglasses: @zeantsoi

File: reddit_adzerk/adzerkpromote.py [hunk 1]
```diff
@@ -111,10 +112,17 @@ def date_from_adzerk(date_str):
...
+        "title": "",
+        "author": "",
+        "target": "",
+        "ecpm": "{{ad.ecpm}}",
+        "priorityId": "{{ad.flight.priorityId}}",
+        "moatQuery": "moatClientLevel1={{ad.flight.advertiserId}}" +
+            "&moatClientLevel2={{ad.creative.id}}" +
+            "&moatClientSlicer1={{site.id}}" +
+            "&moatClientSlicer2={{properties.subreddit}}" +
+            "&zMoatSZ={{ad.creative.width}}x{{ad.creative.height}}",
```
Comment (zeantsoi) [inline reddit_adzerk/adzerkpromote.py]: Just checking that there's actually a `z` in `zMoatSZ`.

**Comment by zeantsoi** [comment]
LGTM, agreed on `priorityId` :fish:

File: reddit_adzerk/adzerkpromote.py (hunk 1 above)
Comment (dwick) [inline reddit_adzerk/adzerkpromote.py]: yeah, that's correct.

**Comment by dwick** [comment]
Add eCPM to ad response and instrument it: bde9389b94293e1318ffee1c785c0d458039262b
Add priority to ad response and instrument it: 23ce87777eb3513d4566edcde3c3358b43be3c07

just the moat stuff remains.

**Pull request merged**